src/main/java/core/basesyntax/services/impl/FruitShopReportGenerator.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.services.impl;
+
+import static core.basesyntax.db.StorageDao.fruitStorage;
+
+import core.basesyntax.services.ReportGenerator;
+import java.util.Map;
+
+public class FruitShopReportGenerator implements ReportGenerator {
+    private static final String COMMA = ",";
+    private static final String HEADER = "fruit,quantity";
+    private StringBuilder builder = new StringBuilder();
```

Comment:
Please make it a local method local variable. This is required because:
1. For example we want to generate report twice, after the first report this variable will be fulfilled with data that will affect the second report
2. If we work in a multithreading environment some threads could override this object and as a result we will have not consistent data